src/tail/session.rs

```diff
@@ -78,7 +85,7 @@ impl Session {
     }
 }
 
-async fn get_tunnel_url() -> Result<String, failure::Error> {
+async fn get_tunnel_url(metrics_port: u16) -> Result<String, failure::Error> {
```

Comment:
this is the last straw. my next refactor will move this tunnel url fetching into the tunnel struct.

Reply:
lol